bounded-collections/src/lib.rs

```diff
@@ -93,7 +93,7 @@ impl<T: Default> Get<T> for GetDefault {
 }
 
 macro_rules! impl_const_get {
-	($name:ident, $t:ty) => {
+	($name:ident, $t:ty, get_into: [$($larger:ty),*]) => {
```

Comment:
You can simplify the implementation and let it do the compiler: https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=a77851b761e90d2246deb9e53395c734